pkgs/development/python-modules/pyluach/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+
+  pytestCheckHook,
+  beautifulsoup4,
+}:
+
+buildPythonPackage rec {
+  pname = "pyluach";
+  version = "2.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "simlist";
+    repo = "pyluach";
+    tag = "v${version}";
+    hash = "sha256-1DmxZsqpRgYaGEiyan6peRwOgQcZ8uuB0nKvRafaCJs=";
```

Comment:
This hash seems to be wrong, you'll need to update this. This can happen because you have the source with the hash that's currently here in your nix store, and nix doesn't download the source in that case.